### DISCUSSION TITLE
Software Engineering Survey on GitHub Discussions.

### DISCUSSION THREAD

**Discussion by raux** [vercel/vercel#5028] [Help]
Do you find GitHub discussions useful as a communication channel? Maybe not? 
Hello Developers! I am Raula Gaikovina Kula, a Software Engineering researcher
https://raux.github.io/

Together with a team of researchers, we are interested in your thoughts about
using the GitHub Discussions Beta. We noticed that your project is currently using.

We have developed a simple 3 question survey that could take up to 5 mins to complete.
https://forms.gle/peBxiio8u99QvKMk6

Thank you in advance for your time and apologize if this is not a proper use of this channel.

@raux
Raula, PhD on behalf of Hideaki, Sebastian, Nicole and Christoph

**Comment by markozxuu** [comment]
Hi @raux  ! 👋  I'm not sure if your message is considered spam. Btw I just finished answering your entire survey : )

<img width="738" alt="Screen Shot 2020-08-13 at 1 59 40" src="https://user-images.githubusercontent.com/16585386/90103930-f948dc00-dd08-11ea-818a-0517cb51b77c.png">
